File: core/model/relationships/dto/NetworkFromPlayerPerspectiveDto.cs
using System.Collections.Generic;

namespace SocialNetwork.DTO
{
    public class NetworkFromPlayerPerspectiveDto
    {
        public string RelationshipId { get; set; }
        public int ?RelationshipStrength { get; set; }
        public List<string> RelationshipTags { get; set; }

        public string PlayerId { get; set; }
        public string PlayerName { get; set; }
        public List<string> PlayerTags { get; set; }
        public List<NetworkFromPlayerPerspectiveDto> Relationships { get; set; }
    }
}

File: Visualization/SPA/src/app/DTO/relationships/NetworkFromPlayerPerspectiveDto.ts
export interface NetworkFromPlayerPerspectiveDto {
  playerEmail: string;
  playerTags: any;
  playerId: string;
  playerName: string;
  emotionalStatus: string;
  relationshipStrength:number;
  relationshipId: string;
  RelationshipTags: string[];
  relationships: NetworkFromPlayerPerspectiveDto[];
}
